function-app/distribution/state.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>3.43.0"
+    }
+  }
+  backend "azurerm" {
+    resource_group_name  = "tfstate"
```

Comment:
I see but it will not be understood for someone who see this in our account. It makes sense to have the tfstate string but we should have some more string that will tell us this is distribution storage account.